src/Microsoft.OData.Client/ALinq/ResourceSetExpression.cs

```diff
@@ -110,5 +110,27 @@ protected override QueryableResourceExpression CreateCloneWithNewTypes(Type newT
                 this.OperationParameters,
                 this.IsAction);
         }
+
+        /// <summary>
+        /// Creates a copy of the current ResourceSetExpression. Object references remain the same.
+        /// </summary>
+        /// <returns>A copy of this</returns>
+        internal override QueryableResourceExpression CreateClone()

```

Comment:
>CreateClone [](start = 54, length = 11)

can this method call the method above?

return CreateClone(this.Type, this.ResourceType)?

Reply:
Or just to call the Above method in the place where to call "CreateClone()" ?

---
In reply to: [396908061](https://github.com/OData/odata.net/pull/1656#discussion_r396908061) [](ancestors = 396908061)

Reply:
I have called method above as suggested.